concent_api/core/message_handlers.py

```diff
@@ -1417,3 +1442,16 @@ def is_message_recieved_in_wrong_state(subtask_id, forbidden_states):
 
 def are_items_unique(items: list):
     return len(items) == len(set(items))
+
+
+def handle_validating_if_list_of_golem_messages_is_signed_with_key(
```

Comment:
please rename it to: `validate_that_golem_messages_are_signed_with_key` ot sth similar

Reply:
Actually I have named it this way on purpose, to show that this function is not actually doing validation, but uses validation as a part of the view and handles its result in view. Do you still think it should be changed ?

Reply:
quite frankly I would still be for renaming - without looking at the function code, I couldn;t figure out what it might be doing ;)